include/grpcpp/ext/channelz_server_plugin.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ * Copyright 2015 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPCPP_EXT_CHANNELZ_SERVER_PLUGIN_H
+#define GRPCPP_EXT_CHANNELZ_SERVER_PLUGIN_H
+
+#include <grpcpp/impl/server_builder_plugin.h>
+#include <grpcpp/support/config.h>
+
+namespace grpc {
+class ServerInitializer;
+class ChannelzServer;
+}  // namespace grpc
+
+namespace grpc {
+namespace channelz {
+
+class ChannelzServerPlugin : public ::grpc::ServerBuilderPlugin {
```

Comment:
I agree with @markdroth that we don't need to expose this plugin. Actually, the src file for this header already adds the plugin automatically. The reflection header this PR is following shouldn't have exposed its plugin either. 

But note that the internal `grpc++_init.cc` is using ` reflection::InitProtoReflectionServerBuilderPlugin()` to "manually" register the plugin. So that file should have access to the init function, although I guess an internal API might be enough.  

BTW, server load reporting service can't be a reference for this because it has some build dependency consideration.

Reply:
Moved this file out of the public API